packages/local_testnet/Readme.md

```diff
@@ -206,40 +231,52 @@ E3_COMPUTE_PROVIDER_BATCH_SIZE=4 # Must be a power of 2
 
 ## Running Ciphernodes
 
-In the root `enclave` directory, you have to run the Ciphernodes. To run 4 Ciphernodes, use the provided script `run_ciphernodes.sh`. Ensure you run the script from the root `enclave` directory to set the environment variables correctly:
+Please make sure that your scripts are run from the `enclave` root directory - this is mandatory to set the environment variables correctly. Depending on your operating system, you may need to give additional execution permissions to each script (using `sudo chmod +x script_name.sh`). The following commands assume that `enclave' and `CRISP' share the same parent folder.
+
+Navigate to the root of the `enclave` repository. To run 3 Ciphernodes, use the provided `run_ciphernodes.sh` script. 
 
 ```sh
-./run_ciphernodes.sh
+./../CRISP/packages/local_testnet/run_ciphernodes.sh
```

Comment:
hmm ouch. I guess this is going to be easier once we migrate CRISP to enclave and setup better tooling.

Reply:
> hmm ouch. I guess this is going to be easier once we migrate CRISP to enclave and setup better tooling. 

yup, it's wordy and ugly from dx pov. At the same time I think making things as explictly as we can is the only way to make the process ugly but effortless

Reply:
We can fix a lot of this sort of stuff through the devx drive

Reply:
100%